### PR TITLE
Add pkg idd

### PR DESCRIPTION
**Pull request by etw33**

### CODE REVIEW

File: packages/idd/idd.0.1/opam [hunk 1]
```diff
@@ -0,0 +1,21 @@
+opam-version: "2.0"
+name: "idd"
+depends: [
+  "ocaml" {>= "4.04.0"}
+  "dune" {build}
+  "base"
+  "ppx_jane"
+  "ppx_inline_test"
```
Comment (mseri) [inline packages/idd/idd.0.1/opam]: Do you need this always or is it just a test dependency?

File: packages/idd/idd.0.1/opam [hunk 2]
```diff
@@ -0,0 +1,21 @@
+opam-version: "2.0"
+name: "idd"
+depends: [
+  "ocaml" {>= "4.04.0"}
+  "dune" {build}
+  "base"
```
Comment (mseri) [inline packages/idd/idd.0.1/opam]: I think you need a lower bound on v1.12 o v1.13. There have been various breaking changes in the releases of base

Comment (etw33) [reply packages/idd/idd.0.1/opam]: Why v1.12 or v1.13 if v1.0 hasn't been released yet?

Comment (mseri) [reply packages/idd/idd.0.1/opam]: Sorry, ot was a typo, I meant 0.

File: packages/idd/idd.0.1/opam [hunk 3]
```diff
@@ -0,0 +1,20 @@
+opam-version: "2.0"
+name: "idd"
+depends: [
+  "ocaml" {>= "4.04.0"}
+  "dune" {build}
+  "base" {>= "0.12.0"}
+  "ppx_jane"
+  "ppx_deriving"
+]
+build: ["dune" "build" "-p" name]
```
Comment (kit-ty-kate) [inline packages/idd/idd.0.1/opam]:
```suggestion
build: ["dune" "build" "-p" name "-j" jobs]
```
you might want to allow users to control the number of jobs they use for the compilation of this package

File: packages/idd/idd.0.1/opam [hunk 4]
```diff
@@ -0,0 +1,20 @@
+opam-version: "2.0"
+name: "idd"
+depends: [
+  "ocaml" {>= "4.04.0"}
+  "dune" {build}
+  "base" {>= "0.12.0"}
```
Comment (kit-ty-kate) [inline packages/idd/idd.0.1/opam]:
```suggestion
  "base" {>= "v0.12.0"}
```
the `v` prefix is non-optional for recent janestreet packages

**Comment by kit-ty-kate** [comment]
Thanks!

**Pull request merged**